jwala-services/src/main/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImpl.java

```diff
@@ -50,17 +53,21 @@
 
     private final BinaryDistributionLockManager binaryDistributionLockManager;
 
+    private final JvmPersistenceService jvmPersistenceService;
+
     public WebServerServiceImpl(final WebServerPersistenceService webServerPersistenceService,
                                 final ResourceService resourceService,
                                 @Qualifier("webServerInMemoryStateManagerService")
                                 final InMemoryStateManagerService<Identifier<WebServer>, WebServerReachableState> inMemoryStateManagerService,
                                 final String templatePath,
-                                final BinaryDistributionLockManager binaryDistributionLockManager) {
+                                final BinaryDistributionLockManager binaryDistributionLockManager,
```

Comment:
Ask the team if you should use class variable auto-wiring since that is what we talked about doing going forward